engines/aws/app/models/aws/vpc.rb

```diff
@@ -6,6 +6,11 @@ class Vpc < AWSResource
     def aws_create
       response = @cli.create_vpc()
       self.id = JSON.parse(response)['Vpc']['VpcId']
+      # update the VPC settings
+      ['--enable-dns-hostnames', '--enable-dns-support'].each do |attr|
+        @cli.modify_vpc_attribute(self.id, attr)
```

Comment:
Do we know if the vpc needs to be fully provisioned before modification? And if so is this waiting for that state to be reached prior to update?

Reply:
great catch, and yes the code should wait for VPC creation to finish, subnets can definitely not be added until VPC is completely setup.

Reply:
The operation of creating a VPC does not take the same time as creating a cluster or other longer operations.
So we could not wait/do not have to wait for VPC in the same way as those resources but, for extra sanity, I will
modify the VPC once is available